doc/guides/feature_compat.md

```diff
@@ -0,0 +1,95 @@
+| empty statements | :x: |  |
+| `__len` metamethod | :heavy_check_mark: |  |
+| `__ipairs` metamethod | :x: |  |
+| `__pairs` metamethod | :heavy_check_mark: |  |
+| `bit32` library | :heavy_check_mark: | Replaces `bit` library, which is still available for compatibility. |
+| `collectgarbage` isrunning, generational, incremental options | :x: | `collectgarbage` does not exist in CC:T. |
+| new `load` syntax | :heavy_check_mark: |  |
+| `loadfile` mode parameter | :heavy_check_mark: | Supports both 5.1 and 5.2+ syntax. |
+| removed `loadstring` | :large_orange_diamond: | Only if `disable_lua51_features` is enabled in the configuration. |
+| removed `getfenv`, `setfenv` | :large_orange_diamond: | Only if `disable_lua51_features` is enabled in the configuration. |
+| `rawlen` function | :heavy_check_mark: |  |
+| negative index to `select` | :heavy_check_mark: |  |
+| removed `unpack` | :large_orange_diamond: | Only if `disable_lua51_features` is enabled in the configuration. |
+| arguments to `xpcall` | :x: |  |
+| second return value from `coroutine.running` | :x: |  |
+| removed `module` | :heavy_check_mark: |  |
+| `package.loaders` -> `package.searchers` | :x: |  |
+| second argument to loader functions | :heavy_check_mark: |  |
+| `package.config` | :heavy_check_mark: |  |
+| `package.searchpath` | :heavy_check_mark: |  |
+| removed `package.seeall` | :heavy_check_mark: |  |
+| `string.dump` on functions with upvalues (blanks them out) | :heavy_check_mark: |  |
+| `string.rep` separator | :x: |  |
+| `%g` match group | :x: |  |
+| removal of `%z` match group | :x: |  |
+| removed `table.maxn` | :large_orange_diamond: | Only if `disable_lua51_features` is enabled in the configuration. |
+| `table.pack`/`table.unpack` | :heavy_check_mark: |  |
+| `math.log` base argument | :heavy_check_mark: |  |
+| removed `math.log10` | :large_orange_diamond: | Only if `disable_lua51_features` is enabled in the configuration. |
+| `*L` mode to `file:read` | :heavy_check_mark: |  |
+| `os.execute` exit type + return value | :x: | `os.execute` does not exist in CC:T. |
+| `os.exit` close argument | :x: | `os.exit` does not exist in CC:T. |
+| `istailcall` field in `debug.getinfo` | :x: |  |
+| `nparams` field in `debug.getinfo` | :heavy_check_mark: |  |
+| `isvararg` field in `debug.getinfo` | :heavy_check_mark: |  |
+| `debug.getlocal` negative indices for varargs | :x: |  |
+| `debug.getuservalue`/`debug.setuservalue` | :x: | Userdata are rarely used in CC:T, so this is not necessary. |
+| `debug.upvalueid` | :heavy_check_mark: |  |
+| `debug.upvaluejoin` | :heavy_check_mark: |  |
+| tail call hooks | :x: |  |
+| `=` prefix for chunks | :heavy_check_mark: |  |
+| yield across C boundary | :heavy_check_mark: |  |
+| removal of ambiguity error | :x: |  |
+| identifiers may no longer use locale-dependent letters | :heavy_check_mark: |  |
+| ephemeron tables | :x: | Weak tables are not supported. |
```

Comment:
I thought weak tables were supported (as long as you set `__mode` before setting the metatable). If not, it's probably worth filing a bug on Cobalt.

Reply:
Ah, I couldn't find it when I was working on 5.2 - just found it, and I'll correct this.